### DISCUSSION TITLE
使用doker安装后 hertzbeat的默认安装位置是什么没有找到

### DISCUSSION THREAD

**Discussion by zxl20000616** [apache/hertzbeat#2212] [Q&A]
使用doker安装后 hertzbeat的默认安装位置是什么没有找到

**Comment by zqr10159** [comment]
![image](https://github.com/apache/hertzbeat/assets/30048352/69aa7cfa-f381-4103-bd03-97a9fbe9fbbc)

**Reply by zxl20000616** [reply]
docker run -d -p 1157:1157 -p 1158:1158 --name hertzbeat apache/hertzbeat
我直接用的这个命令  在opt里面没有找到

**Reply by zqr10159** [reply]
目录需要挂载出来，才能在宿主机上修改文件
